src/utils.js

```diff
@@ -180,6 +181,64 @@ export function buildLocaleDateAndTime(dateTime, timezone, locale) {
     .format('L LT');
 }
 
+export function getSelectedSlipData(pickSlipsData, selectedRequestId) {
+  const sel = pickSlipsData.filter((pickSlip) => {
```

Comment:
If `filter` does not find any element that fits your condition it returns empty array [] instead of _undefined_.
As a result `if (sel === undefined)` never happen.
See the `filter` [description](https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/Array/filter).

Reply:
`if (sel === undefined) ` seems happening. lets discuss.